config/rds/config.go

```diff
@@ -57,6 +57,19 @@ func Configure(p *config.Provider) {
 			"MasterUserSecretInitParameters":     "ClusterMasterUserSecretInitParameters",
 			"MasterUserSecretObservation":        "ClusterMasterUserSecretObservation",
 		}
+		desc, _ := comments.New("If true, the password will be auto-generated and"+
+			" stored in the Secret referenced by the masterPasswordSecretRef field.",
+			comments.WithTFTag("-"))
+		r.TerraformResource.Schema["auto_generate_password"] = &schema.Schema{
+			Type:        schema.TypeBool,
+			Optional:    true,
+			Description: desc.String(),
+		}
+		r.InitializerFns = append(r.InitializerFns,
+			common.PasswordGenerator(
+				"spec.forProvider.masterPasswordSecretRef",
+				"spec.forProvider.autoGeneratePassword",
+			))
```

Comment:
It would be great to add a description for `master_password` like here: https://github.com/upbound/provider-aws/blob/fb0fb486e6225cdab27a447c48cb36f98464884e/config/docdb/config.go#L43

Reply:
Thanks for pointing it out. I've added the master_password description.